cmp/grid/GridModel.js

```diff
@@ -255,8 +261,10 @@ export class GridModel {
         useVirtualColumns = false,
         autosizeOptions = {},
         experimental,
+        enableColChooser,
         ...rest
     }) {
+        apiDeprecated(enableColChooser, 'enableColChooser', "Use 'colChooserModel' instead");
```

Comment:
think we want this *not* to appear as an argument, and call apiRemoved(rest.enableColChooser, )